test/PreAMMBatcher-e2e.ts

```diff
@@ -133,23 +130,31 @@ describe("PreAMMBatcher-e2e", () => {
   };
 
   beforeEach(async () => {
-    token0 = await deployContract(walletDeployer, ERC20, ["token0", "18"]);
-    token1 = await deployContract(walletDeployer, ERC20, ["token1", "18"]);
-    uniswapFactory = await deployContract(walletDeployer, UniswapV2Factory, [
-      walletDeployer.address,
+    const PreAMMBatcher = await ethers.getContractFactory("PreAMMBatcher");
+
+    token0 = await waffle.deployContract(walletDeployer, ERC20, [
+      "token0",
+      "18",
+    ]);
+    token1 = await waffle.deployContract(walletDeployer, ERC20, [
+      "token1",
+      "18",
```

Comment:
I believe this should mean that, unlike with Truffle, contract artifacts won't be storing the address they were deployed at during testing.

Reply:
Correct. This should be the same as before. The wrapped `waffle.deployedContract` just uses the `buidler` MockProvider instead of having to create a new one.